migrate.py

```diff
@@ -613,7 +615,9 @@ def format_change_element(change_element):
 
 def _gh_username(username, users, gh_auth):
     try:
-        return users[username]
+        user = users[username]
+        return user if user else None  # No GH account for this user
```

Comment:
Would `user = users.get(username)` be the same thing?

And why do this? We're already catching the `KeyError` and passing...

Reply:
We're explicitly converting a user-provided empty-string to `None` and short-circuiting, to allow the caller to exclude usernames that shouldn't be mapped to GH. Falling through via `pass` would wind up in the auto-associate logic instead.

The goal is to let the caller disable the auto-associate behavior if it's wrong. If a BB user has a name that also exists on GH, but isn't the same person, there's no way currently to disable that mapping.